src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "moviesessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @ManyToOne
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
+
+    public MovieSession() {
```

Comment:
Don't create a default constructor when it is not needed. In this case, you have a parameterized constructor, and Hibernate is capable of using it for proxy creation. If you're not using the default constructor elsewhere, it should be removed.